repo-transfer.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Transfer a repository to the W3C GitHub organization</title>
+    <link rel="stylesheet" href="css/wgio.css">
+    <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico">
+  </head>
+  <body>
+    <header>
+      <h1>Transfer a repository to the W3C GitHub organization</h1>
+    </header>
+<nav>
+  <a href="/">Home</a>
+  •
+  <a href="https://github.com/w3c/">Repositories</a>
+  •
+  <a href="https://help.github.com/">GitHub Help</a>
+</nav>
+
+<main>
+  <p>This document describes how to transfer a repository to the W3C GitHub organization.</p>
+  <p>General information about repository transfer on GitHub is available <a href="https://docs.github.com/en/github/administering-a-repository/transferring-a-repository">here</a>.</p>
+  <ol>
+    <li>Make sure the <a href="https://github.com/w3c/">W3C GitHub organization</a> doesn't already have a repository with the same name. If the name is already taken, the owner will need to rename the repository first
+    <li>The owner of the repository requests the transfer from the repository settings<br>
```

Comment:
In most cases, it will be more complicated than that: you can only transfer a repo to an organization in which you have owner rights - for the w3c org, this means someone from the staff.

So what's usually needed when transferring a repo from a non-staff user account to w3c is to first transfer it to someone from the staff, and then have that person transfer it.

For a repo coming from another organization, it may be necessary to invite someone from the staff to join as at least an admin on the source repo - this happens regularly with repos being transfered from wicg to w3c, but I'm not entirely clear if the requirement is admin privs or owner privs

Reply:
I was actually expecting an email to the owners to approve the transfer but after testing, it just triggers an error.
I updated the PR to detail the different use cases.